docs/whatsnew/5.0.rst

```diff
@@ -20,15 +20,14 @@ files and converting between different Python objects.
 The methods use Astropy's Unified I/O registry so custom formats can be
 registered. Details are discussed in an addition to the docs.
 
-Currently no file formats are registered, but the syntax is as follows:
+Currently the only registered read/write format is "ascii.ecsv", like for Table.
 
 .. doctest-skip::
 
-    >>> from astropy.cosmology import Planck18
-    >>> Planck18.write('<file name>.<format>')
+    >>> from astropy.cosmology import Cosmology, Planck18
+    >>> Planck18.write('<file name>.ecsv')
     >>>
-    >>> from astropy.cosmology import Cosmology
-    >>> cosmo = Cosmology.read('<file name>.<format>')
+    >>> cosmo = Cosmology.read('<file name>.ecsv')
     >>> cosmo == Planck18
     True
 
```

Comment:
I think the examples below are a bit long for the What's New. The idea is not to comprehensively show the capabilities but just highlight things enough for interested people to click through to the docs link. I think each What's New entry should fit comfortably on one laptop screen (and maybe half the people are reading on mobile?). So maybe just:
```
Transformations between ``Cosmology`` and `dict`, ``QTable``,  and``Model`` are
pre-registered, enabling for example::

    >>> from astropy.cosmology import Cosmology, Planck18
    >>> cm = Planck18.to_format("mapping")
    >>> cm
    {'cosmology': <class 'astropy.cosmology.flrw.FlatLambdaCDM'>,
     'name': 'Planck18',
     'H0': <Quantity 67.66 km / (Mpc s)>,
     'Om0': 0.30966,
     ...

For details see :ref:`whatever`.

Reply:
SGTM. I would like to show Table and Model. I think those are the compelling examples.